main.py

```diff
@@ -51,7 +52,17 @@ def verify_user(email_addr, secure_string):
     else:
         return False
 
-# Function registers the user by adding data into the db and sending him a mail for authrntication via EBIA
+# TODO addressed: Handles cases where the user is already verified.
+def already_verified(email_addr):
+    # Get value from the database
+    val = db.get(email_addr)
+    # Check if the email address in the database has already been verified
+    if val['confirmed'] == True:
+        return True
+    else:
+        return False
+
+# Function registers the user by adding data into the db and sending them a mail for authrntication via EBIA
```

Comment:
Add comments to functions following style as here mentioned [here](https://www.python.org/dev/peps/pep-0257/).